lib/freeclimb/utils/signature_information.rb

```diff
@@ -0,0 +1,45 @@
+require 'date'
+require 'openssl'
+require 'cgi'
+
+module Freeclimb
+    class SignatureInformation
+        attr_accessor :requestTimestamp
+        attr_accessor :signatures
+
+        def initialize(requestHeader)
+            @requestTimestamp = 0
+            @signatures = []
+            signatureHeader = requestHeader.try(:split, ",")
+            signatureHeader.each { |signature|
+                header = signature.try(:split, "=").at(0)
+                value = signature.try(:split, "=").at(1)
```

Comment:
why do multiple splits?
you can easily destructuring instead. 
```
header, value = signature.try(:split, "=")
```

Reply:
will change in latest commit